core/src/main/java/tc/oc/pgm/controlpoint/ControlPoint.java

```diff
@@ -414,10 +417,13 @@ private void dominate(Competitor dominantTeam, Duration dominantTime) {
       // Point is owned and must go through the neutral state before another team can capture it
       if (dominantTeam == this.controllingTeam) {
         // owner is recovering the point
-        recover(dominantTeam, dominantTime);
+        recover(dominantTeam, dominantTime, contested);
```

Comment:
For this one i'm not totally sure if contested *can* be removed, because specifically for recovering towards a neutral state both dominant and controlling would be null, and it could be coming from a situation where the point is contested or not contested. 
However, i also believe in no point would it ever make a difference, because in that situation the hill would simply fall-back to neutral, and after reaching neutral, it would in no case be able to fall to a contested scenario.

So i'm pretty confident that contested can be removed here too, and simply recurse with a `false` because there is no situation where that can make a difference.